apps/kyc-server/README.md

```
@@ -2,158 +1,87 @@
-The `App` component provides the navigation UI, and the router renders different content based on the current URL.
-
-## Development
-
-### Prerequisites
-
-- Bun 1.2.4 or higher
-
-### Setup
+## Setup
 
+1. Install dependencies:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)